src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -97,6 +103,23 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
   background-color: var(--pf-c-data-toolbar--BackgroundColor);
 }
 
+// Divider
+.pf-c-data-toolbar__content-section,
+.pf-c-data-toolbar__group {
+  // set this var here so specificity is 20
+  > .pf-c-divider {
```

Comment:
why is `pf-c-divider` set to `--pf-c-data-toolbar--c-divider--m-vertical--spacer` but below `.pf-c-divider.pf-m-vertical` is set to `--pf-c-data-toolbar--spacer`

Reply:
`--pf-c-data-toolbar--spacer` is a utility variable. It isn't designed to define strict values, rather it is meant to be manipulated, but as a consequence it's very sensitive to specificity. So I set `--pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar--c-divider--m-vertical--spacer);` on `__group` and `__content-section` to retain a specificity of  20, so that in can later be overwritten by a modifier or by a custom theme. Doing this allows for consumers to override the base spacer value `--pf-c-data-toolbar--c-divider--m-vertical--spacer` for the vertical spacer AND use `data-toolbar` s modifiers as if it belonged to the component. 

Because it has no assignment in horizontal/default `pf-c-divider` the variable is ignored until it's called in `pf-m-vertical`